quay-enterprise/installing-contributor-extensions-postgres.md

```diff
@@ -0,0 +1,26 @@
+# Installing Contributor Extensions for Postgres
+
+As of release 2.1.0 Quay Enterprise requires the [pg_trgm](https://www.postgresql.org/docs/current/static/pgtrgm.html) extension when using Postgres. This extension ships as part of the [Additional Supplied Modules](https://www.postgresql.org/docs/current/static/contrib.html). The requirements to allow extensions to be installed can be satisfied via package management tools on Debian and RedHat-based systems.
```

Comment:
s/2.1.0/2.1.0,/
s/Postgres/PostgreSQL/
s/ships as/is/
replace last sentence: "These modules can be installed with standard package management tools on Debian and RedHat-based systems."